README.md

```diff
@@ -72,12 +72,4 @@ upcoming Docs Italia.
 
     This will compile static assets and watch files required for the theme to reload at runtime.
 
-6. Update modernizr.js
-
-  ```
-  ./node_modules/.bin/modernizr -uc ./modernizr.json -d ./js/modernizr.min.js
-  ```
-  
-  This will compile modernizr.js starting form the configuration present in ./modernizr.json
```

Comment:
Non so se anche questo va ripristinato dopo il rollback di Modernizer

Reply:
No perché la build di Modernizr è automatizzata in Grunt, questa sarebbe la versione manuale.